rpcpb/rpc.proto

```diff
@@ -206,7 +206,7 @@ message StartRequest {
   // Must be non-empty to install custom VMs.
   optional string plugin_dir = 6;
 
-  // The map of custom VM name to its genesis file path.
+  // The list of custom VM name, its genesis file path, and (optional) subnet id to use.
```

Comment:
Here too, it's a list of BlockhainSpec.... If you don't want to change all these it's fine

Reply:
I prefer not for this